staging/src/k8s.io/cli-runtime/pkg/resource/helper.go

```diff
@@ -38,6 +38,9 @@ type Helper struct {
 	RESTClient RESTClient
 	// True if the resource type is scoped to namespaces
 	NamespaceScoped bool
+	// If true, then use server-side dry-run to not persist changes to storage
+	// for verbs and resources that support server-side dry-run.
```

Comment:
Add a note maybe to mention that this should only be used against dry-run enabled server?

Reply:
Done.